ews/src/types/common.rs

```diff
@@ -263,6 +263,19 @@ pub struct CopyMoveItemData {
     pub return_new_item_ids: Option<bool>,
 }
 
+/// The common format for folder move and copy operations.
+#[derive(Clone, Debug, XmlSerialize)]
+pub struct CopyMoveFolderData {
+    /// The destination folder for the copied/moved folder.
+    ///
+    /// See <https://learn.microsoft.com/en-us/exchange/client-developer/web-service-reference/tofolderid>
+    pub to_folder_id: BaseFolderId,
+    /// The identifiers for each folder to copy/move.
```

Comment:
```suggestion
    pub to_folder_id: BaseFolderId,

    /// The identifiers for each folder to copy/move.
```

nitpick: We should have a line break between documented fields for readability.